packages/stack/src/views/Stack/Card.tsx

```diff
@@ -517,7 +517,8 @@ export default class Card extends React.Component<Props> {
           // Make sure that this view isn't removed. If this view is removed, our style with animated value won't apply
           collapsable={false}
         />
-        <View pointerEvents="box-none" {...rest}>
+        {/* Make sure this view is not removed on the new architecture, as it causes focus loss during navigation on Android. */}
+        <View pointerEvents="box-none" collapsable={false} {...rest}>
```

Comment:
```suggestion
        <View
          pointerEvents="box-none"
          // Make sure this view is not removed on the new architecture, as it causes focus loss during navigation on Android.
          // This can happen when the view flattening results in different trees - due to `overflow` style changing in a parent.
          collapsable={false}
          {...rest}
        >
```